src/lib.rs

```diff
@@ -1089,6 +1089,19 @@ mod test {
         let _c: u8 = Standard.sample(&mut r);
     }
 
+    #[test]
+    // #[cfg(feature="stc")]
+    fn test_rng_rc_refcell_trait() {
+		use std::ops::Deref;
+        use distributions::{Distribution, Standard};
+        // let rng = rng(110);
+        let mut r = & *::std::rc::Rc::new(::std::cell::RefCell::new(rng(110)));
```

Comment:
We would not require the `& *` here if we had an actual impl for `Rc<RefCell<R>>` but navigating rustc to a `&mut &RefCel<R>` is tricky.